config/local.yml

```diff
@@ -11,6 +11,8 @@ elasticsearch:
   - localhost
   cluster: foxtrot
   tableNamePrefix: foxtrot
+  countQueryTimeout: 10000
```

Comment:
Maybe create a map of opcode to timeout. That way adding timeout for other operations would be easier.

Reply:
opcode is understood by foxtrot. From ES query perspective there are largely two categorisation, aggregation and query_then_fetch. Hence I have kept two.